stdlib/tkinter/ttk.pyi

```diff
@@ -1071,26 +1083,22 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
     @overload
     def tag_bind(self, tagname: str, *, callback: str) -> None: ...
     @overload
-    def tag_configure(self, tagname: str, option: Literal["text"]) -> str: ...
-    @overload
-    def tag_configure(self, tagname: str, option: Literal["image"]) -> str: ...
-    @overload
-    def tag_configure(self, tagname: str, option: Literal["anchor"]) -> tkinter._Anchor | Literal[""]: ...
-    @overload
     def tag_configure(self, tagname: str, option: Literal["foreground", "background"]) -> tkinter._Color: ...
     @overload
-    def tag_configure(self, tagname: str, option: str) -> Any: ...
+    def tag_configure(self, tagname: str, option: Literal["font"]) -> _FontDescription: ...
+    @overload
+    def tag_configure(self, tagname: str, option: Literal["image"]) -> str: ...
     @overload
     def tag_configure(
         self,
         tagname: str,
         option: None = ...,
         *,
-        text: str = ...,
-        image: tkinter._ImageSpec = ...,
-        anchor: tkinter._Anchor = ...,
-        background: tkinter._Color = ...,
+        # There is also 'text' and 'anchor', but they don't seem to do anything, using them is likely a bug
         foreground: tkinter._Color = ...,
+        background: tkinter._Color = ...,
+        font: _FontDescription = ...,
+        image: tkinter._ImageSpec = ...,
     ) -> _TreeviewTagDict | None: ...
```

Comment:
Can you do the same changes to `_TreeviewTagDict`?

Reply:
Good catch! Done.